cpu/numactl.py

```diff
@@ -67,6 +72,57 @@ def setUp(self):
 
         build.make(self.sourcedir)
 
+        self.iface = self.params.get("interface", default="")
+        self.ping_count = self.params.get("ping_count", default=100)
+        self.peer = self.params.get("peer_ip", default="")
+        interfaces = netifaces.interfaces()
+        if not self.iface:
+            self.cancel("Please specify interface to be used")
+        if self.iface not in interfaces:
+            self.cancel("%s interface is not available" % self.iface)
+        if not self.peer:
+            self.cancel("peer ip should specify in input file")
+        self.ipaddr = self.params.get("host_ip", default="")
+        self.netmask = self.params.get("netmask", default="")
+        self.localhost = LocalHost()
+        self.networkinterface = NetworkInterface(self.iface, self.localhost)
+        try:
+            self.networkinterface.add_ipaddr(self.ipaddr, self.netmask)
```

Comment:
if ipaddr and netmask are not valid are we handling this scenario as if run this script without having this value it will prompt me as i did wrong

Reply:
@PraveenPenguin filed separate PR for this feature support, PR #5541